pkg/agent/bakerapi.go

```diff
@@ -85,6 +87,32 @@ func (agentBaker *agentBakerImpl) GetNodeBootstrapping(ctx context.Context, conf
 	return nodeBootstrapping, nil
 }
 
+func (agentBaker *agentBakerImpl) GetNodeBootstrappingForSelfContained(
+	ctx context.Context,
+	config *datamodel.NodeBootstrappingConfiguration,
+) (*datamodel.NodeBootstrapping, error) {
+	// TODO: add windows support
+	if config.AgentPoolProfile.IsWindows() {
+		return agentBaker.GetNodeBootstrapping(ctx, config)
+	}
+	config.Version = "v0"
+	nodeBootstrapping, err := agentBaker.GetNodeBootstrapping(ctx, config)
+	if err != nil {
+		return nil, err
+	}
+	configJSON, err := json.Marshal(config)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal nbc, error: %w", err)
+	}
+	//nolint: lll
```

Comment:
what is `lll`

Reply:
The line with bash script is too long, but I don't want to split it across multiple lines.